var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -43,7 +43,6 @@ class Pango(AutotoolsPackage):
 
     depends_on("pkg-config", type="build")
     depends_on("harfbuzz")
-    depends_on("cairo")
```

Comment:
Why was this change necessary? With this change, if I run:
```
$ spack spec pango ^cario@1.2.3
```
it will complain that "pango does not depend on cairo". This is a bug in the concretization algorithm. This line was the current workaround. Same for `gtkplus`.